esp-hal/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `time::current_time` API (#1503)
 - ESP32-S3: Add LCD_CAM Camera driver (#1483)
 - `embassy-usb` support (#1517)
+- Added `fmt-packages` xtask subcommand (#1551)
```

Comment:
Does this deserve a esp-hal changelog entry?? I think I would just `skip-changelog`

Reply:
It does not, I am not sure why I need to keep repeating this but:

__Changes to the `esp-hal/CHANGELOG.md` file are only required if there are user-facing changes to the `esp-hal` package__.